menu_vypocty.py

```diff
@@ -0,0 +1,34 @@
+def menu():
+    print("\nZvolte jednu z možností:")
+    print("Q - Quit (Ukončit program)")
+    print("S - Square (Umocnit číslo na druhou)")
+    print("D - Double (Vynásobit číslo dvěma)")
+    print("H - Half (Vydělit číslo dvěma)")
+    print("P - Print (Zobrazit hodnotu čísla n)")
+
+# Hlavní část programu
+n = float(input("Zadejte číslo: "))

```

Comment:
If it's not the intention, I think it would be a good idea to add the option to change the value of "n" after each cycle, otherwise the user will be working with the same value indefinitely.

Reply:
I think that the intention is to manipulate `n` in `while` loop.
But having file documentation in top saying it would be nice.